apps/web/src/app/(default)/page.tsx

```diff
@@ -1,212 +1,189 @@
-          >
-            <h2 className="text-center text-2xl font-medium md:text-3xl">Bedriftspresentasjoner</h2>
-
-            <ArrowRight className="ml-2 inline h-6 w-6 transition-transform group-hover:translate-x-2" />
-          </Link>
-          <hr />
-          {bedpresses.length > 0 ? (
-            <ul>
-              {bedpresses.map((bedpres) => {
-                return (
-                  <li key={bedpres._id}>
-                    <HappeningPreview happening={bedpres} />
-                  </li>
-                );
-              })}
-            </ul>
-          ) : (
-            <div className="py-4">
-              <p className="text-balance text-center text-xl font-medium">
-                Ingen kommende bedriftspresentasjoner for øyeblikket
+            </Reveal>
+          </div>
+
+          <div className="grid gap-16 sm:grid-cols-2">
+            <Reveal translateX={-200}>
+              <h2 className="mb-4 text-xl font-medium">Arrangementer</h2>
+
+              {events.length > 0 ? (
+                <ul>
+                  {events.map((event) => (
+                    <li key={event._id} className="flex items-center justify-between gap-2">
+                      <Link href={createHappeningLink(event)} className="truncate hover:underline">
+                        {event.title}
+                      </Link>
+                      <p className="text-nowrap text-muted-foreground">
+                        {format(new Date(event.date), "EEE. dd.MM", {
+                          locale: nb,
+                        })}
+                      </p>
+                    </li>
+                  ))}
+                </ul>
+              ) : (
+                <p>Det er ingen kommende arrangementer</p>
+              )}
+            </Reveal>
+
+            <Reveal translateX={200}>
+              <h2 className="mb-4 text-xl font-medium">Bedriftpresentasjoner</h2>
+
+              {bedpresses.length > 0 ? (
+                <ul>
+                  {bedpresses.map((bedpres) => (
+                    <li key={bedpres._id} className="flex items-center justify-between gap-2">
+                      <Link
+                        href={createHappeningLink(bedpres)}
+                        className="truncate hover:underline"
+                      >
+                        {bedpres.title}
+                      </Link>
+                      {/* <p className="text-nowrap text-muted-foreground">ti. 02.04</p> */}
+                      <p className="text-nowrap text-muted-foreground">
+                        {format(new Date(bedpres.date), "EEE. dd.MM", {
+                          locale: nb,
+                        })}
+                      </p>
+                    </li>
+                  ))}
+                </ul>
+              ) : (
+                <p>Det er ingen kommende bedriftspresentasjoner</p>
+              )}
+            </Reveal>
+          </div>
+
+          <div className="grid gap-16 md:grid-cols-2">
```

Comment:
Tror det ser litt bedre ut hvis denne er på samme linje som hva er echo istedenfor echo logoen.

Reply:
Flyttet! Bare å pushe masse greier nå om du vil, eller etter den er merget. Synes ikke at mangel på bilder burde være en blocker for å kunne spice opp designet litt.